packages/doenetml-worker-rust/lib-doenetml-core/src/core/state_var_interfaces/boolean_state_var_interfaces.rs

```diff
@@ -244,6 +257,12 @@ impl SingleDependencyBooleanStateVarInterface {
     }
 }
 
+impl From<SingleDependencyBooleanStateVarInterface> for StateVar<bool> {
```

Comment:
I think we should get rid of all the `SingleDependency` stuff until we know we need it.